pgpubsub/listen.py

```diff
@@ -0,0 +1,43 @@
+from functools import wraps
+
+import pgtrigger
+
+from pg_notify.channel import CustomPayloadChannel, TriggerPayloadChannel
+from pg_notify.notify import Notify
+
+
+def listen(channel_name: str=None):
+    def _listen(callback):
+        CustomPayloadChannel.register(callback, channel_name)
+        @wraps(callback)
+        def wrapper(*args, **kwargs):
+            return callback(*args, **kwargs)
+        return wrapper
+    return _listen
+
+
+def post_insert_listen(model, channel_name=None):
+    return trigger_listen(
+        model,
+        trigger=Notify(
+            name=channel_name,
+            when=pgtrigger.After,
+            operation=pgtrigger.Insert,
+        ),
+        channel_name=channel_name,
+    )
+
+
+def trigger_listen(
+    model,
+    trigger,
+    channel_name: str=None,
+):
+    def _trig_listen(callback):
+        TriggerPayloadChannel.register(callback, channel_name)
+        trigger.install(model)
```

Comment:
I believe you can do this https://django-pgtrigger.readthedocs.io/en/latest/tutorial.html#configuring-triggers-on-external-models

i.e. `pgtrigger.register(TriggerInstance)(ModelClass)` to register it

When triggers are registered like this, django-pgtrigger will handle installing them during migrations. Although I exposed the `.install` interface on a trigger, it was generally not meant for users to use it directly